### ISSUE TITLE
Vile stand not on level 15

### ISSUE THREAD

**Issue by Azezzara** [lantus/devilution-nx#29]
I reached level 15 and have explored the entire level and i can't find the vile stand which holds the Lazarus staff.
Without this the game is unbeatable and you can't progress.
Is it a bug or am i missing something?

**Comment by Azezzara** [comment]
[single_0.zip](https://github.com/lantus/devilution-nx/files/4433454/single_0.zip)
Here is the save file. it only works to load on a switch not pc

**Comment by AJenbo** [comment]
Please use the up stream release, this fork is unmaintained, and produce invalid save games amongst other issues.
https://github.com/diasurgical/devilutionX/releases

**Comment by Azezzara** [comment]
does this mean my save game is ruined and i have to start a new game?

**Comment by AJenbo** [comment]
Yes, I have talked to @lantus about updating the fork but he hasn't had time.
